### DISCUSSION TITLE
[Feature] Method to convert annotated images to COCO panoptic dataset

### DISCUSSION THREAD

**Discussion by cclo-astri** [wkentaro/labelme#1125] [Ideas / Feature requests]
Hi ,

I could use the python provided (`instance_segmentation/labelme2coco.py`) to convert my own images to MS COCO format and import to detectron2 for training and eval. 

I wonder if there exists a example script to convert the annotated images to COCO panoptic dataset format.

Please help. 

Thanks.

**Comment by EMCP** [comment]
please append a link to this panoptic format you're talking about

**Comment by EMCP** [comment]
also, did you get that script cited `labelme2coco.py` to work for you ? 

I am seeing errors

**Comment by cclo-astri** [comment]
@EMCP , @deepseek 

COCO Panoptic format
[http://cocodataset.org/#format-data](url)

Corresponding API repo
[https://github.com/cocodataset/panopticapi](url)

But as I am a newbie of python, I really need some help to make use of these APIs.

Thanks.

**Comment by wshilaji** [comment]
Have you solved this problem？can you teach me?

**Comment by cclo-astri** [comment]
@wshilaji 

No, I still waiting for solution. 

Sorry.

**Comment by wshilaji** [comment]
> @wshilaji
> 
> No, I still waiting for solution.
> 
> Sorry.

I'm learning cocoapi and the file format of coco panoptic dataset.
I think I can solve this problem this week
If successful, I will send you my python file script

**Comment by ShengkaiWu** [comment]
@wshilaji Can you convert annotated images to COCO panoptic datasets?

**Comment by wshilaji** [comment]
> @wshilaji Can you convert annotated images to COCO panoptic datasets?

https://github.com/wshilaji/panoptic_coco_scipt     star  it  Thanks
